front-end-notes/css-attributes-and-selectors.md

```diff
@@ -0,0 +1,51 @@
+#CSS Selectors
+
+### Selectors
+
+A CSS selector is the part of a CSS rule set that actually selects the content you want to style.
+
+```
+.red-background { ## This is the selector.
+  /* CSS rule goes here */
+}
+```
+
+### Selector Specificity
+
+Specificity is the means by which a browser decides which CSS property values are the most relevant to an element and therefore will be applied. For instance, instead of using `p`, which selects all `p` elements, the selector can be made more specific by indicating which p element should be selected: `p.awesomeClass`.
+
+**Warning:** Using the `*` universal selector can lead to styling being applied to unwanted elements, making code hard to maintain and debug.
+
+# HTML Attributes
+
+Attributes provide additional information about HTML elements. For example, an <code>href</code> attribute provides address information for links. They are always specified in the **start tag**, i.e. the beginning tag of the HTML element. For example: `<a href="www.imanattributevalue.com"></a>`.
+
+### CSS Attribute Selectors
+
+When selecting an element with a specific attribute in CSS, the attribute name and value should be surrounded in square brackets and the value should be in quotes. For example:
+```
+li[id="thirdListItem"] {
+  /* CSS property-value pairs go here. */
+}
+```
+
+#Helpful Links
+
+Here are a few of our most helpful cheat-sheets that have LOTS of good information should you get stuck.
+
+### CSS SELECTORS
+

```

Comment:
I think we were told at the beginning of the cohort that w3schools links are awful for reference? Perhaps provide an alternate reference than W3 schools?